### ISSUE TITLE
Don't allow players to tow locked vehicles

### ISSUE THREAD

**Issue by sethduda** [sethduda/AdvancedTowing#1]

**Comment by sethduda** [comment]
Also, provide a variable that mission creators can set to allow towing of locked vehicles.

**Comment by ka0sCODE** [comment]
Check config.cpp

```
class Lock: ExileAbstractAction
			{
				title = "Lock";
				condition = "((locked ExileClientInteractionObject) isEqualTo 0) && ((locked ExileClientInteractionObject) != 1)";
				action = "true spawn ExileClient_object_lock_toggle";
			};

			// Unlocks a vehicle
			class Unlock: ExileAbstractAction
			{
				title = "Unlock";
				condition = "((locked ExileClientInteractionObject) isEqualTo 2) && ((locked ExileClientInteractionObject) != 1)";
				action = "false spawn ExileClient_object_lock_toggle";
			};
```

As well as ExileClient_object_vehicle_network_lockResponse.sqf / ExileClient_object_lock_toggle.sqf, / ExileClient_object_vehicle_network_lockVehicleRequest.sqf. 

Easy identifier: 

```
if (_mode) then
{ 
_vehicleObject lock 2;
}
else 
{
_vehicleObject lock 0;
};
```

Should be fairly easy to add checks for this.

**Comment by sethduda** [comment]
Thanks - added to the latest source

**Comment by sethduda** [comment]
Released on steam in v1.2.1
